src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerSkill.java

```diff
@@ -54,6 +76,9 @@ public class CaseWorkerSkill implements Serializable {
             insertable = false, updatable = false, nullable = false)
     private CaseWorkerProfile caseWorkerProfile;
 
+    @Fetch(FetchMode.SUBSELECT)
+    @OneToMany(targetEntity = Skill.class, mappedBy = "caseWorkerSkill", cascade = ALL, orphanRemoval = true)
```

Comment:
Hi @shivajihmcts on further discussions with the Team, the above skill relationship in caseworker skills should be many to one Request you to please do the needful Thank you so much ... FYI @SujeethKumarAri @kavithagade12

Reply:
Hi Sahitya
Please check code on the staff-ui-changes, its have same mapping as you expecting, Please take a update from branch. Its a part of create staff profile , mapping is covered into it